src/core/js/scriptLoader.js

```diff
@@ -30,7 +30,8 @@
                 a11y: 'libraries/jquery.a11y',
                 scrollTo: 'libraries/scrollTo',
                 bowser: 'libraries/bowser',
-                'enum': 'libraries/enum'
+                enum: 'libraries/enum',
+                jqueryMobile: 'libraries/jquery.mobile.custom'
```

Comment:
can we try something like this 
``Modernizr.load([
            {
                test: Modernizr.touch,
                yes: 'libraries/jquery.mobile.custom.js'
            }
        ]);``
if it is overkill instance, no problem 👍